src/molecules/walletBalanceList/types.ts

```diff
@@ -0,0 +1,15 @@
+export interface TableItem {
+  id: number;
+  token: any;
+  tokenName: string;
+  balance: string;
+  wrappedBalance: string;
+  inUse?: boolean;
+}
+
+export interface WalletBalanceList {
+  type: 'dashboard' | 'checkout';
+  wbData: any;
```

Comment:
@goshatravin @shan8851  

i suggest we start taking response types from swagger and creating types for data we get, what do you guys think?

Here are links to swagger:

https://api.uniwhales.io/swagger/
https://api-staging.uniwhales.io/swagger/ (in case new endpoint was pushed its gonna be first on staging)

Reply:
Makes sense @ArinCantCode , so I assume there should already be a response type to match this in swagger? i can see there is a bunch of web socket data, that may be where it lives. Thanks for linking me to Swagger. I will go through now and the get the correct response type.

Reply:
@kosotd  are you able to help with providing example response for this wallet balance data - it is going to be in the checkout flow? I did have a look around in Swagger, but I could not seem to find it.